packages/table/src/cell/cell.tsx

```diff
@@ -105,10 +107,24 @@ export class Cell extends React.Component<ICellProps, {}> {
         // note: these aren't actually used by truncated format, just in shouldComponentUpdate
         const modifiedChildren = React.Children.map(this.props.children, child => {
             if (style != null && React.isValidElement(child)) {
-                return React.cloneElement(child as React.ReactElement<any>, {
-                    parentCellHeight: style.height,
-                    parentCellWidth: style.width,
-                });
+                const childType = child.type;
+                // can't get prototype of "string" child, so treat those separately
+                if (typeof child === "string" || typeof childType === "string") {
+                    return child;
+                } else {
+                    const isTruncatedFormat =
+                        childType.prototype === TruncatedFormat.prototype ||
+                        TruncatedFormat.prototype.isPrototypeOf(childType) ||
+                        childType.prototype === JSONFormat.prototype ||
+                        JSONFormat.prototype.isPrototypeOf(childType);
```

Comment:
I could see adding a utility function `isElementType()`--we do this sort of thing in a few places throughout the codebase. But not a big deal for now, at least it's well-formatted.